exercises/list-ops/list_ops_test.rb

```diff
@@ -0,0 +1,107 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'list_ops'
+
+class ListOpsTest < Minitest::Test
+  def test_count_empty
+    assert_equal 0, ListOps.arrays([])
+  end
+
+  def test_count_normal
+    skip
+    assert_equal 5, ListOps.arrays(Array.new(5))
+  end
+
+  def test_count_gigantic
```

Comment:
I think this test is not adding/testing a new functionality, maybe it can be removed? Was this a problem during implementation?